gtests/net/mptcp/regressions/close_mpj_timeout2.pkt

```diff
@@ -0,0 +1,37 @@
+// connection initiated by packetdrill
```

Comment:
(maybe best to rename this file? Because "2" is not really explicit, we need to check the code and the `diff` with the other one to know the difference)

Reply:
ok, will do